nvme.pyx

```diff
@@ -154,7 +154,7 @@ cdef class NvmeDevice(object):
         Preempt the existing `pr_key` with `cur_key`
         '''
         opcode = nvme.nvme_op_codes.nvme_cmd_resv_acquire
-        reservation_acquire_action = (2 & 0x7)
+        reservation_acquire_action = (1 & 0x7)
         reservation_acquire_type = (1 << 8)
```

Comment:
Similarly
```
# See 7.4, Figure 394 Reservation Type Encoding
ctypedef enum nvme_resv_type:
    nvme_resv_type_write_exclusive = 0x01
    nvme_resv_type_exclusive_access = 0x02
    nvme_resv_type_write_exclusive_registrants_only = 0x03
    nvme_resv_type_exclusive_access_registrants_only = 0x04
    nvme_resv_type_write_exclusive_all_registrants = 0x05
    nvme_resv_type_exclusive_access_all_registrants = 0x06
```

then
`reservation_acquire_type = (nvme_resv_type_write_exclusive << 8)`

Reply:
Oh, that should have been section 7.2 again in the comment.